packages/core/entity/FileDetail/RenderZarrThumbnailURL.ts

```diff
@@ -103,10 +103,18 @@ export async function renderZarrThumbnailURL(zarrUrl: string): Promise<string |
                 // Determine a slice of the image that has the best chance at being a good thumbnail.
                 // X and Y will have full range, while Z will be the middle integer.
                 // Non-spatial dims will default to None.
-                const axes = transformAxes(multiscales[0].axes);
+                const defaultAxes = [
+                    { name: "t", type: "time" },
+                    { name: "c", type: "channel" },
+                    { name: "z", type: "space" },
+                    { name: "y", type: "space" },
+                    { name: "x", type: "space" },
+                ];
+                const axes = transformAxes(multiscales[0].axes || defaultAxes);
```

Comment:
Do you happen to have any test files that exhibit this behavior? would love to experiment with some!